packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -125,29 +110,134 @@ class FullCalendar extends React.Component {
+          <Menu.Button look="raised">
+            <Menu.Item>
+              <Menu.ButtonGroup
+                look="raised"
+                melt
+              >
+                <Menu.Item>
+                  <Button
+                    onClick={() => {
+                      this.calendar.prev()
+                      this.handleRangeChange()
+                    }}
+                    icon="chevron-left"
+                    title={this.msg('client.scheduler.previous')}
+                  />
+                </Menu.Item>
+                <Menu.Item>
+                  <Button
+                    look="raised"
+                    onClick={() => {
+                      this.calendar.today()
+                      this.handleRangeChange()
+                    }}
+                  ><FormattedMessage id="client.scheduler.today"/></Button>
+                </Menu.Item>
+                <Menu.Item>
+                  <Button
+                    onClick={() => {
+                      this.calendar.next()
+                      this.handleRangeChange()
+                    }}
+                    icon="chevron-right"
+                    title={this.msg('client.scheduler.next')}
+                  />
+                </Menu.Item>
+              </Menu.ButtonGroup>
+            </Menu.Item>
+            <Menu.Item>
+              <Button
+                icon={this.props.isLoading ? '' : 'sync'}
+                look="raised"
+                onClick={() => { if (!this.props.isLoading) { this.props.onRefresh() } }}
+                pending={this.props.isLoading}
+                title={this.msg('client.scheduler.reload')}
+              />
+            </Menu.Item>
+            <Menu.Item>{title}</Menu.Item>
+            <Menu.Item>
+              <Menu.ButtonGroup
+                look="raised"
+                melt
+              >
+                <Menu.Item>
+                  <Button
+                    {...(this.getButtonInkProps('timelineDay'))}
+                    onClick={() => this.changeView('timelineDay')}
+                  >
+                    <FormattedMessage id="client.scheduler.day"/>
+                  </Button>
+                </Menu.Item>
+                <Menu.Item>
+                  <Button
+                    {...(this.getButtonInkProps('timelineWeek'))}
+                    onClick={() => this.changeView('timelineWeek')}
+                  >
+                    <FormattedMessage id="client.scheduler.week"/>
+                  </Button>
+                </Menu.Item>
+                <Menu.Item>
+                  <Button
+                    {...(this.getButtonInkProps('timelineMonth'))}
+                    onClick={() => this.changeView('timelineMonth')}
+                  >
+                    <FormattedMessage id="client.scheduler.month"/>
+                  </Button>
+                </Menu.Item>
+              </Menu.ButtonGroup>
+            </Menu.Item>
+          </Menu.Button>
```

Comment:
I thint we could move this whole menu to a component called Header, what do you think?